refactoring/initial_global.py

```diff
@@ -19,43 +19,41 @@ def add_person(name, age, job, relations):
     }
     group[name] = new_person
 
-
-group = {
-    "Jill": {
-        "age": 26,
-        "job": "biologist",
-        "relations": {
-            "Zalika": "friend",
-            "John": "partner"
-        }
-    },
-    "Zalika": {
-        "age": 28,
-        "job": "artist",
-        "relations": {
-            "Jill": "friend",
-        }
-    },
-    "John": {
-        "age": 27,
-        "job": "writer",
-        "relations": {
-            "Jill": "partner"
+if __name__ == "__main__":
+    group = {
+        "Jill": {
+            "age": 26,
+            "job": "biologist",
+            "relations": {
+                "Zalika": "friend",
+                "John": "partner"
+            }
+        },
+        "Zalika": {
+            "age": 28,
+            "job": "artist",
+            "relations": {
+                "Jill": "friend",
+            }
+        },
+        "John": {
+            "age": 27,
+            "job": "writer",
+            "relations": {
+            "   Jill": "partner"
+            }
```

Comment:
Nice! Now the group won't be loaded if you imported the file which means you're not including uncessary variables in your module